src/services/formatting/smartIndenter.ts

```diff
@@ -279,7 +279,14 @@ module ts.formatting {
 
         function getActualIndentationForListItem(node: Node, sourceFile: SourceFile, options: EditorOptions): number {
             let containingList = getContainingList(node, sourceFile);
-            return containingList ? getActualIndentationFromList(containingList) : Value.Unknown;
+
+            if (containingList) {

```

Comment:
I think that handling just list elements is not enough. In cases like the one below if you hit Enter at the marker indentation will be incorrect because currently there are no elements in the list so current logic will not be hit.
```typescript
Promise
    .resolve
    .then(/**/
        );
```